scripts/zkai_d64_stwo_vector_row_surface_probe.py

```
@@ -350,12 +363,138 @@ def expected_issue_scope() -> dict[str, Any]:
     }
 
 
+def proof_public_instance_contract(fixture: dict[str, Any]) -> dict[str, Any]:
+    statement = fixture["statement"]
+    binding = fixture["commitments"]
+    public_instance = FIXTURE.public_instance_payload(binding)
+    public_instance_commitment = FIXTURE.public_instance_commitment(binding)
+    if public_instance_commitment != statement["public_instance_commitment"]:
+        raise D64VectorRowSurfaceError("fixture public-instance commitment mismatch")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)